examples/tutorials/01_basics/07_backpack_lights.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2016 Anki, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License in the file LICENSE.txt or at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Control Cozmo's Backpack lights
+
+This script shows how you can control Cozmo's backpack lights and set
+them to different colors.
```

Comment:
For kids, I think it would be fun in this script to list in a comment all the colors that his lights can be set to (although maybe not all colors are exposed in SDK)? I thought it was fun in the Scratch prototype that the backpack can be set to many different colors.